com.trelby.Trelby.yaml

```diff
@@ -0,0 +1,43 @@
+app-id: com.trelby.Trelby
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: trelby
+finish-args:
+  - --socket=cups
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --talk-name=org.freedesktop.FileManager1
+  - --filesystem=home
+  - --filesystem=/tmp
+
+modules:
+  - name: flatpak-xdg-utils
```

Comment:
why do you need this?

Reply:
Because I couldn't at the time figure out how to get xdg-open to work, I think I'll try again and remove that. Sorry.

Reply:
it is there in the runtime. `xdg-open` just work